docs/api-reference/mapbox/mapbox-layer.md

```diff
@@ -101,6 +101,7 @@ Parameters:
 - `props` (Object)
   + `props.id` (String) - an unique id is required for each layer.
   + `props.deck` (`Deck`, optional) - a `Deck` instance that controls the rendering of this layer. If provided, the layer will be looked up from its layer stack by `id` at render time, and all other props are ignored.
+  + `props.viewId` (String, optional) - If provided, the view will be looked up from the `Deck` instance's view stack by `id` at render time. Intended to only be used when a `Deck` instance is provided that has multiple `views`. If it is not provided or cannot be found, the first view will be used instead.
```

Comment:
If the prop is not provided, then we should keep the current behavior (only render a single viewport from Map).

If the prop is provided but no matching view is found, then we shouldn't replace any viewport.

Reply:
That is how it will behave. Should I replace the doc with your comment's phrasing?